sql/step_a111_settings.sql

```diff
@@ -0,0 +1,88 @@
+-- Drop the existing table
+DROP TABLE IF EXISTS OH_SETTINGS;
+
+-- Create settings table structure
+CREATE TABLE OH_SETTINGS (
+  SETT_ID INT(11) NOT NULL AUTO_INCREMENT COMMENT 'Default generated ID',
+  SETT_CODE VARCHAR(50) NOT NULL,
+  SETT_CATEGORY ENUM('general', 'telemetry', 'printer', 'dicom', 'examination', 'report', 'sms') NOT NULL DEFAULT 'general',
```

Comment:
let's use the same grouping already defined in the `settings.properties.dist` in GUI
and then reflect in the `INSERT INTO` below